lang/interpret_test.go

```diff
@@ -1322,9 +1332,25 @@ func TestAstFunc2(t *testing.T) {
 			}
 
 			iast, err := ast.Interpolate()
-			if err != nil {
+			if (!fail || !fail8) && err != nil {
 				t.Errorf("test #%d: FAIL", index)
-				t.Errorf("test #%d: interpolate failed with: %+v", index, err)
+				t.Errorf("test #%d: Interpolate failed with: %+v", index, err)
+				return
+			}
+			if fail8 && err != nil {
+				// TODO: %+v instead?
+				s := fmt.Sprintf("%s", err) // convert to string
```

Comment:
`err.Error()`? I think that's what fmt.Sprintf() does here anyway

Reply:
Sure, why not =D Not doing the %+v it seems anyways.